arduino-ide-extension/src/browser/arduino-preferences.ts

```diff
@@ -241,6 +241,14 @@ export const ArduinoConfigSchema: PreferenceSchema = {
       ),
       default: false,
     },
+    'arduino.checkForUpdates': {
+      type: 'boolean',
+      description: nls.localize(
+        'arduino/preferences/checkForUpdate',
+        "Configure whether you receive automatic updates for the IDE, boards, and libraries. Requires an IDE restart after change. It's true by default."
```

Comment:
```suggestion
        "Receive notifications of available updates for the IDE, boards, and libraries. Requires an IDE restart after change. It's true by default."
```

The reason for the proposal to change "automatic updates" -> "notifications of available updates" is that "automatic updates" gives me the impression these things will be updated completely automatically, which is not the case.

The reason for the proposal to remove "Configure whether you" is because I didn't like the significant increase in text that resulted from my other proposal, so I looked for something that could be cut from the sentence.

Reply:
Thanks for the proposal.

> Requires an IDE restart after change

This must be a manual update. Later we can make it automatic, but for now, it should be restarted by the users. Are you OK with the _Requires an IDE restart after change._ working?

Reply:
>Are you OK with the Requires an IDE restart after change. wording?

Yes, I'm perfectly fine with that. Thanks.